Rakefile

```diff
@@ -1,7 +1,12 @@
 begin
-  require 'rspec/core/rake_task'
+  require "rspec/core/rake_task"
   RSpec::Core::RakeTask.new(:spec)
-rescue LoadError
+rescue LoadError # rubocop:disable Lint/SuppressedException
 end
```

Comment:
I don't think we need this cautionary behaviour of begin and rescue here as we'd only expect this rake file to be run in a scenario when we have rspec available (unlike deployable web apps where they'd not have rspec in the production group).

Be nice if we can load rubocop in the same way too.